packages/devtools_app/lib/src/screens/network/network_model.dart

```diff
@@ -180,4 +182,60 @@ class Socket extends NetworkRequest {
 
   @override
   int get hashCode => id.hashCode;
+
+  SocketStatistic get socketData => _socket;
+
+  static Socket fromJson(Map<String, Object?> json) {
+    return Socket(
+      SocketStatistic.parse(json['socket'] as Map<String, dynamic>)!,
```

Comment:
use `Object?` instead of `dynamic`

Also, please put these string key values in consts or in the enum below

Reply:
This comment still stands. Also, can you use a factory constructor here instead?

Reply:
done.